website/content/docs/nia/cli/index.mdx

```diff
@@ -82,4 +82,10 @@ consul-terraform-sync task disable -port=2000 task_a
 
 Below are options that can be used across all commands:
 
-- `-port=<port>` - (int: 8558) The port that the Consul-Terraform-Sync daemon serves its API.
+- `-port=<port>` - (int: 8558) The port that the Consul-Terraform-Sync daemon serves its API. This port is used with the address `http://localhost`. If a different scheme or address is required, use `-http-addr` instead.
+- `-http-addr=<address>` - (string: http://localhost:8558) The address and port of the Consul-Terraform-Sync API. The value can be an IP address or DNS address. This can also be specified via the `CTS_HTTP_ADDR` environment variable.
+- `-ssl-verify` - (bool: true) Whether to verify TLS/SSL connections to the Consul-Terraform-Sync API. The default value is `true`, as disabling verification is a potential security vulnerability. This can also be specified using the `CTS_SSL_VERIFY` environment variable.
```

Comment:
maybe something about how although it is default set to true it won't affect `http`/non-secure connections?